docs/users_guide/framework_basics/the_affine_space.md

```diff
@@ -238,70 +355,201 @@ Taxi distance:   31.2544 km
 !!! note
 
     It is not allowed to subtract two point origins defined in terms of `absolute_point_origin`
-    (e.g. `mean_sea_level - mean_sea_level`) as those do not contain information about the unit
-    so we are not able to determine a resulting `quantity` type.
+    (e.g., `home - home`) as those do not contain information about the unit, so we are not able
+    to determine a resulting `quantity` type.
 
 
-### Temperature support
+### Relative _point_ origin
 
-Another important example of [relative point origins](#relative-point-origins) is support
-of temperature quantity points in units different than kelvin [`K`].
+We often do not have only one ultimate "zero" point when we measure things.
 
-The [SI](../../appendix/glossary.md#si) system definition in the **mp-units** library provides
-two predefined point origins:
+For example, let's assume that we have the following absolute point origin:
 
 ```cpp
-namespace mp_units::si {
+constexpr struct mean_sea_level : absolute_point_origin<mean_sea_level, isq::altitude> {} mean_sea_level;
+```
 
-inline constexpr struct absolute_zero : absolute_point_origin<absolute_zero, isq::thermodynamic_temperature> {} absolute_zero;
-inline constexpr struct ice_point : relative_point_origin<absolute_zero + 273.15 * kelvin> {} ice_point;
+If we want to model a trip to Mount Everest, measuring all daily hikes from the `mean_sea_level`
+might not be efficient. We may know that we are not good climbers, so all our climbs can be
+represented with an 8-bit integer type, allowing us to save memory in our database of climbs.
 
-}
+For this purpose, we can define a `relative_point_origin` in the following way:
+
+```cpp
+constexpr struct everest_base_camp : relative_point_origin<mean_sea_level + 5364 * m> {} everest_base_camp;
 ```
 
-With the above, we can be explicit what is the origin of our temperature point. For example, if
-we want to implement the degree Celsius scale we can do it as follows:
+The above can be used as an origin for subsequent _points_:
 
 ```cpp
-using Celsius_point = quantity_point<isq::Celsius_temperature[deg_C], si::ice_point>;
+constexpr quantity_point first_climb_alt = everest_base_camp + isq::altitude(std::uint8_t{42} * m);
+static_assert(first_climb_alt.quantity_from(everest_base_camp) == 42 * m);
+static_assert(first_climb_alt.quantity_from(mean_sea_level) == 5406 * m);
 ```
 
+As we can see above, the `quantity_from()` member function returns a relative distance from the
+provided point origin.
+
+
+### Converting between different representations of the same _point_
+
+As we might represent the same _point_ with _vectors_ from various origins, the **mp-units** library
+provides facilities to convert the _point_ to the `quantity_point` class templates expressed in
+terms of different origins.
```

Comment:
```suggestion
provides facilities to convert the _point_ to `quantity_point` class templates expressed in
terms of origins relative to each other in the type system.
```
I remember "different" raising concerns about "reinterpreting origins".
So this makes it more clear up front rather than leaving it all to the note at the end.